rpc/internal/perf_service.go

```diff
@@ -22,13 +22,15 @@ import (
 )
 
 type perfService struct {
-	env cedar.Environment
+	env                                    cedar.Environment
+	performanceAnalysisProxyServiceCreator func(model.PerformanceAnalysisProxyServiceOptions) perf.PerformanceAnalysisProxyService
```

Comment:
Since we are now within the scope of the perf service, we do not need to be so explicit. We can call this field `analysisProxyClientCreator`.